src/main/resources/assets/computercraft/lua/rom/modules/main/http/pastebin.lua

```diff
@@ -0,0 +1,128 @@
+--- Parse the pastebin code from the given code or URL
+local function parseCode(paste)
+    local patterns = {
```

Comment:
Now that I'm looking at this, might be worth hoisting this out of the function definition. Sorry.